### PR TITLE
 Show DNA if available

### PR DESCRIPTION
**Pull request by chienwei-lan**
DNA PR

### CODE REVIEW

**Comment by chienwei-lan** [comment]
http://jira.xilinx.com/browse/PR-8479

File: src/runtime_src/driver/include/xclhal2.h [hunk 1]
```diff
@@ -138,6 +138,7 @@ struct xclDeviceInfo2 {
   unsigned short mVccIntVol;
   unsigned short mVccIntCurr;
   unsigned short mNumCDMA;
+  char mDNA[128];
```
Comment (sonals) [inline src/runtime_src/driver/include/xclhal2.h]: We cannot change this struct since it breaks binary compatibility.

**Comment by chienwei-lan** [comment]
get value with sysfs_get(subdev, sysfs, errmsg, &string) for short term, will implement key-value pair API in the coming week.

**Comment by chienwei-lan** [comment]
image

**Comment by chienwei-lan** [comment]
It will look like...
image

**Pull request merged**